tests/comprehension_dict.py.uast

```diff
@@ -1,5 +1,6 @@
-Status:  ok
+Status:  error
```

Comment:
Oh f*ck I was so centered on the non zero column number and offset that skipped these ones, good catch, thanks.